docs/sources/tempo/operations/monitor/set-up-monitoring.md

```diff
@@ -12,45 +12,45 @@ If you don't have a cluster available, you can use the linked documentation to s
 
 To set up monitoring, you need to:
 
-* Use Grafana Agent Flow or Grafana Alloy to remote-write to Tempo and set up Grafana to visualize the tracing data by following [Set up a test app](https://grafana.com/docs/tempo/latest/setup/set-up-test-app/).
-* Update your Grafana Agent Flow configuration to scrape metrics to monitor for your Tempo data.
+* Use Grafana Alloy collector to remote-write to Tempo and set up Grafana to visualize the tracing data by following [Set up a test app](https://grafana.com/docs/tempo/<TEMPO_VERSION>/setup/set-up-test-app/).
+* Update your collector configuration to scrape metrics to monitor for your Tempo data.
```

Comment:
Are you referring to collectors in general? Or just to Alloy? If it's juts Alloy, then we could name it here instead of using `collector`.

Reply:
Will do. This tarted off as a doc to use both Agent Flow and Alloy. We changed it to just Alloy. This is a good catch. 
```suggestion
* Update your Alloy configuration to scrape metrics to monitor for your Tempo data.
```